CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
```

Comment:
Sorry @benSlaughter - can we remove the test changes from the changelog? (We try to keep the changelog for user-facing changes only)